journalpump/senders/google_cloud_logging.py

```diff
@@ -61,7 +66,14 @@ def send_messages(self, *, messages, cursor):
 
         for message in messages:
             msg_str = message.decode("utf8")
-            msg = json.loads(msg_str)
+            # This might not measure exactly 256K but should be
+            # a good enough approximation to handle this error.
+            # In anycase, we are handling this error in the
+            # exception handler below also.
+            if len(message) > GoogleCloudLoggingSender._LOG_ENTRY_QUOTA:
+                msg = { "MESSAGE": "Log entry cannot be logged because its size is greater than GCP logging quota of 256K" }
```

Comment:
the message is simply decoded as a JSON. AFAIK the scheme does not have required fields, so there is not guarantee about how we can/should parse the JSON.
That said, we can do a "best effort" and split the "MESSAGE" field only.
I did it this way because it was an acceptable trade of when I talked to Julien Clarysse